h2o-algos/src/main/java/hex/glm/GLMUtils.java

```diff
@@ -30,6 +30,12 @@
     }
     return gamColIndices;
   }
+  
+  public static GLM.GLMGradientInfo copyginfo(GLM.GLMGradientInfo ginfo) {
```

Comment:
camel case won't hurt readability here: `copyGInfo`

Reply:
yes, I will change it.

Reply:
Done